### DISCUSSION TITLE
Found a strange SFL address

### DISCUSSION THREAD

**Discussion by DominicTing** [sunflower-land/sunflower-land#2279] [General]
There has been a lot of newly minted SFL recently, I checked the liquidity reward pool and found that this address has been receiving a similar amount of SFL. https://polygonscan.com/token/0xd1f9c58e33933a993a3891f8acfe05a68e1afc05?a=0xb62f3f8ecc7b7f18533a4d3b8b331b56506cc75c
The SFL owned by the superior address cannot tell where it came from.

**Comment by SuperSynapse** [comment]
I'm not sure what you were trying to do, but you posted the contract for the SFL token itself, not an individual or wallet or mint amount...

If you want to see who holds SFL, I'm including that link below. Most of SFL is held in the burn address, or contracts such as liquidity providing/etc.

https://polygonscan.com/token/0xd1f9c58e33933a993a3891f8acfe05a68e1afc05#balances
